artifacts_test.py

```diff
@@ -124,6 +125,21 @@ def check_scylla(self):
         self.run_cassandra_stress("write n=10000 -mode cql3 native -pop seq=1..10000")
         self.run_cassandra_stress("mixed duration=1m -mode cql3 native -rate threads=10 -pop seq=1..10000")
 
+    def check_cqlsh(self):
+        output = self.node.run_cqlsh("desc keyspaces")
```

Comment:
shouldn't it be inside some try except statement? what if we fail to run CQL commands?

Reply:
it's inside a subTest, wouldn't that be enough ? i.e. it would continue the test, but the test would fail (as it should be, if cqlsh is broken, exactly why i'm adding this check)

Reply:
good enough only when inside a context manager... approved